cmd/kubelet/app/server.go

```diff
@@ -207,6 +208,14 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				klog.Fatal(err)
 			}
 
+			if kubeletConfig.KubeletCgroups == "" && kubeletConfig.KubeReservedCgroup != "" {
```

Comment:
This isn't quite what I meant.  I meant that someone may be intentionally relying on the default for `--kubelet-cgroups`, and also setting `--kube-reserved-cgroups`.   I don't think we need this first section at all.  I think it would be confusing to change the kubelet cgroup.